InfoLogger/test/lib/services/mocha-query-service.test.js

```diff
@@ -205,6 +206,39 @@ describe(`'QueryService' test suite`, () => {
       );
     });
 
+    it('should be able to fill in a prepared statement and log it when executing a query', async () => {
+      const sqlDataSource = new QueryService(config.mysql);
+      sqlDataSource._logger = {
+        debugMessage: sinon.stub(),
+      };
+      sqlDataSource._pool = {
+        query: sinon.stub().resolves([{ hostname: 'test', severity: 'W' }]),
+      };
+      await sqlDataSource.queryFromFilters(realFilters, { limit: 10 });
+      const completeSqlQuery = "SELECT * FROM `messages` WHERE `timestamp`>='1563794601.351' AND" +
+        " `timestamp`<='1563794661.354' AND `hostname` = 'test' AND NOT(`hostname` = 'testEx' AND" +
+        " `hostname` IS NOT NULL) AND `severity` IN ('D','W') ORDER BY `TIMESTAMP` LIMIT 10;";
+      assert.strictEqual(sqlDataSource._logger.debugMessage.calledWith(`SQL to execute: ${completeSqlQuery}`), true);
+      // Test the processPreparedSQLStatement() function individually.
```

Comment:
As you are testing only the individual function, it would be better to move the test to its own test file, following the directory structure as well. (`test/utils/mocha-preparedStatementParser.js` and test both functions from that file